resources/event-bus/values.yaml

```diff
@@ -5,11 +5,11 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   event_bus:
-    dir: develop/
-    version: 9e7cfa58
+    dir:
```

Comment:
@suleymanakbas91 What should this `event-bus` point to?
we don't have images in the root directory, no?

Reply:
@montaro For release builds the images are not in `develop/`. Please refer https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/3099/pre-rel08-kyma-components-event-bus/1105128449070075904

Reply:
@montaro here is how we use these values inside a chart: https://github.com/kyma-project/kyma/blob/master/resources/event-bus/charts/publish-knative/templates/deployment.yaml#L29. Of course, we have images in the root directory: https://console.cloud.google.com/gcr/images/kyma-project/EU/event-bus-publish-knative

Reply:
@suleymanakbas91 this is what I really mean 
https://console.cloud.google.com/gcr/images/kyma-project/EU/event-bus?gcrImageListsize=30